### ISSUE TITLE
Output logits for generation

### ISSUE THREAD

**Issue by mayankjobanputra** [explosion/curated-transformers#311]
For calculating perplexities logits are required. I couldn't find any way to do it currently. Maybe there's already a way, please point me to it.

**Comment by shadeMe** [comment]
Thanks for the request! If you're referring to the logits generated by the causal LM, there is currently no way to access them through the `Generator` interface. We'll look into exposing this in a future release.
